buildSrc/build.gradle

```diff
@@ -93,7 +93,7 @@ dependencies {
 
   api 'commons-codec:commons-codec:1.12'
   api 'org.apache.commons:commons-compress:1.19'
-
+  api 'org.apache.ant:ant:1.10.8'
```

Comment:
Can we not implicitly rely on the version bundled with Gradle or is that filtered out from the buildsrc classpath?

Reply:
the ant dependencies are available via `shadow.org.apache.tools.*` which urged me to use the explicit dependency rather than a shadowed one.